website/content/docs/nia/configuration.mdx

```diff
@@ -186,7 +186,7 @@ task {
   - `min` - (string: "5s") The minimum period of time to wait after changes are detected before triggering related tasks.
   - `max` - (string: "20s") The maximum period of time to wait after changes are detected before triggering related tasks. If `min` is set, the default period for `max` is 4 times the value of `min`.
 - `condition` - (obj) The requirement that, when met, triggers CTS to execute the task. When unconfigured, the default condition is to trigger the task on changes in the services configured in [`services`](#services). Only one `condition` may be configured per task. CTS supports different types of conditions, which each have their own configuration options. See [Task Condition](#task-condition) configuration for full details on configuration options for each condition type.
-- `source_input` - (obj) Specifies a Consul object containing values or metadata to be provided to the Terraform Module. When source input is empty, source input will be determined by the condition or services list. Only one `source_input` may be configured per task. CTS supports different types of source input, each source input has their own configuration options. The source input block is currently only supported with [schedule condition](#schedule-condition). See [Task Source Input](#task-source-input) configuration for full details on configuration options for each source input type.
+- `source_input` - (obj) Specifies a Consul object containing values or metadata to be provided to the Terraform Module. The `source_input` block defines any extra source inputs needed for task in addition to any source input provided by the `condition` block or `services` list. Multiple `source_input` blocks can be configured per task. [Task Source Input](#task-source-input) configuration for full details on usage and restrictions.
```

Comment:
```suggestion
- `source_input` - (obj) Specifies a Consul object containing values or metadata to be provided to the Terraform Module. The `source_input` block defines any extra source inputs needed for task execution.  This is in addition to any source input provided by the `condition` block or `services` list. Multiple `source_input` blocks can be configured per task. [Task Source Input](#task-source-input) configuration for full details on usage and restrictions.
```
The original sentence is a bit confusing. Hoping the change is still true to the orginal intent.

Reply:
Yes, thank you. This still keeps the original intent